.github/workflows/release-workflow.yml

```diff
@@ -131,7 +133,7 @@ jobs:
           name: linux-artifacts
           path: |
             nexus-staging/
-            samm-cli-linux.tar.gz
+            samm-cli-${{ github.event.inputs.release_version }}-linux-x86_64.tar.gz
```

Comment:
Can we use `$release_version` here too?

Reply:
Yes, sure. But here we have to make separate variable, I decided to don't this. I will add this.

Thanks!